README.md

```diff
@@ -26,9 +26,12 @@ all objects by the server and is not editable by the client applications. While
 _anything_ within their objects, this property will always tell the Truth. The details are in the
 documentation, but broadly, you will find:
 
-* `created`  specific creation date for this \[version of this] object
-* `isOverwritten`  specific date (if any) this version was updated without versioning
+* `@context`   the RERUM API JSON-LD context file for these terms
+* `alpha`  internal flag for RERUM API version control 
```

Comment:
It may be time for us to drop this flag

Reply:
I mean, from responses. We can keep it internally if we like.

Reply:
I agree, but it is there right now, so we need an explainer for it.  Let's make a ticket for it.